docs/DeveloperGuide.adoc

```diff
@@ -721,62 +721,167 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+| `* * *` | Customer manager | find a customer by name | locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+| `* * *` | Customer manager | Sort customers by name | Locate customers easily
+
+| `* * *` | Customer Manager | Sort customers by tag | Group my customers according to specific criteria, like &#39;paid&#39;, &#39;new&#39; etc
+
+| `* * *` | Clumsy customer manager | Receive a warning when I might potentially make a grave mistake, ie. delete a wrong contact | I am less likely to make mistakes
+
+| `* * *` | Customer manager who values security | Login securely | Keep my customer&#39;s data secure
+
+| `* * *` | Customer Manager | List all my customers | Get a List of all my customers
+
+| `* * *` | Customer Manager | Sort customers by tag | Group my customers according to specific criteria, like &#39;paid&#39;, &#39;new&#39; etc
+
+| `* * *` | Customer manager | Edit information | Update my customers&#39; information when they have changes.
+
+| `* * *` | Customer manager who cannot  remember commands correctly | Have an &#39;auto-complete&#39; feature while typing commands | Be more efficient and reduce the time required to recall the whole command
+
+| `* * *` | Customer manager with customers with multiple emails, addresses and phone numbers. | Add all of their details | Contact them easily in case they don&#39;t respond via one of their usual contact details.
+| `* * *` | Forgetful customer manager | List all tags | I can identify which tags I have, and possibly any near duplicates like &#39;frequent&#39; and &#39;frequent customer&#39;
+
+| `* *` | Customer manager who prioritizes convenience | Be able to access my most frequently / recent searched customers easily | Save time
+
+| `* *` | Customer manager who needs to send letters out | View their addresses on a map | Look-up their address easily
+
+| `* *` | Customer manager | Have alert for customers&#39; birthdays | We (the business) can celebrate with the customer or send out a greeting.
+
+| `* *` | Clumsy customer manager | Undo my command   | Undo my mistake
+
+| `* *` | Customer manager | Clear my address book | Start a new address book again
+
+| `* *` | Customer manager | Profile pictures for my contacts | Verify the identity of the customer
+
+| `* *` | Customer manager | Get customers&#39; bank accounts number | Transfer them money quickly, or check if they transferred us money
+
+
+| `* *` | Customer manager using the app for the first time | Import existing customers from an existing csv file | Do not have to go through the trouble of importing my customer details one by one
+
+| `* *`  | Customer manager | Merge duplicate contacts | Don&#39;t have to change each user manually
```

Comment:
i feel this user story can be broken down into two: 1) merge duplicate contacts of the same user (consider how to implement this, e.g. keep both phone number, email?) 2) change multiple user at the same time